src/core/server/index.ts

```diff
@@ -216,6 +216,9 @@ export {
   UiSettingsServiceSetup,
   UiSettingsServiceStart,
   UserProvidedValues,
+  ImageValidation,
+  DeprecationSettings,
+  StringValidation,
```

Comment:
Ideally now that these are public, and as they are very generic name with very specific usages, they should start with the `UiSettings` prefix to avoid conflicts.